mitodl/micromasters
# Pass object ids to Celery tasks instead of objects

#### What are the relevant tickets?
Part of #2718 

#### What's this PR do?
Refactors tasks to use object ids instead of objects, to avoid possible race conditions and serialization problems, especially when deleting objects. 

#### How should this be manually tested?
Update your profile then go to the learners page to verify that the update indexed properly.

#### Any background context you want to provide?
The next PR will remove the deprecated logic which is only needed for Celery tasks which are still in the queue during the deploy.
